include/unifex/task.hpp

```diff
@@ -386,7 +414,7 @@ struct _task<T>::type : _task_base {
   using awaiter = typename _awaiter<promise_type, OtherPromise>::type;
 
   explicit type(coro::coroutine_handle<promise_type> h) noexcept
-    : coro_(h) {}
+    : coro_holder(h) {}
 
   template<typename Promise>
   friend awaiter<Promise> tag_invoke(tag_t<unifex::await_transform>, Promise&, type&& t) noexcept {
```

Comment:
Doesn't the `await_transform` method need to be updated to pull the coroutine handle out of the new `coro_holder`?
e.g. it's still referencing `t.coro_` below but that data-member has been removed.

Also the awaiter it's constructing is expecting a typed coroutine handle but now you only have a `coroutine_handle<void>` coroutine handle strored in `coro_holder`.

Reply:
Ah, I see - the data-member from the base class is protected so you're still able to access it here.
Never mind.

Reply:
Ah; that was absolutely not intentional.  I'll take this change.